idb/postgres/internal/schema/setup_postgres.sql

```diff
@@ -117,3 +117,6 @@ CREATE TABLE IF NOT EXISTS account_app (
 
 -- For looking up existing app local states by account
 CREATE INDEX IF NOT EXISTS account_app_by_addr_partial ON account_app(addr) WHERE NOT deleted;
+
+-- For looking up app box storage
+{{.AppBox}};
```

Comment:
this has been used as a source file to generate the string in setup_postgres_sql.go. With your template approach, you're already creating the sql string in AppBoxMigration. This makes me wonder why we need the sql file at all.  Maybe texttosource generator should be refactored so each table is assigned to an exported variable so that `const SetupPostgresSql=fmt.Sprintf("%s\n%s\n%s", Table1,Table2,Table3)`
@winder